src/runner.ts

```diff
@@ -420,4 +441,29 @@ export class ResolveRunner implements Types.IResolveRunner {
       .fragment('')
       .toString();
   }
+
+  private isFile(ref: uri.URI): boolean {
+    const scheme = ref.scheme();
+
+    if (scheme === 'file') return true;
+
+    if (!scheme) {
+      // if no scheme set, and ref starts with a '/', assume it's a file
```

Comment:
What if it starts with a drive letter, i.e. `C:\\`?

Reply:
Covered - https://github.com/stoplightio/json-ref-resolver/pull/57/files#diff-02f2832865abf89619d47f8c83164733R137